members.html

```diff
@@ -0,0 +1,49 @@
+---
+layout: page
+title: Members
+description: Regular Attendees of OpenHack Syracuse
+---
+<!-- Main Content -->
+
+<div class="row">
+    {% for member in site.data.members %}
+    <div class="col-lg-3 col-lg-offset-1 col-md-5 col-md-offset-1">
+      <h4 style="text-align: center;">{{member.name}}</h4>
+      <img src="{{member.img}}" style="float: center; height: 160px; margin: 0 auto; border-radius: 10%;" />
+
+      <div style="text-align: center;">
+          {% if member.social.github %}
+          <a href="http://www.github.com/{{member.social.github}}"><i class="fa fa-github"></i></a>
```

Comment:
I believe github/twitter/facebook all default to https. Lets reflect that in the links.